### DISCUSSION TITLE
Loratap double curtain calibration

### DISCUSSION THREAD

**Discussion by timewolf** [Koenkk/zigbee2mqtt#20830] [Devices]
hi,

i've bought a loratap double courtain touch switch like this 

<img width="287" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/44172693/43bcef87-5eab-4265-abf6-b8a220773c68">

zigbee2mqtt detect as https://www.zigbee2mqtt.io/devices/TS130F_dual.html but the calibration method seems not ok.

How can I calibrate that device?

thanks

Fabio

**Comment by guydvir2** [comment]
<img width="1376" height="483" alt="image" src="https://github.com/user-attachments/assets/9818b3c2-427f-4af0-8596-c012cc9ab543" />

simple
